src/back/models/Personnel.ts

```diff
@@ -11,6 +13,21 @@ export type PersonnelAttributes = TeamMember & {
   created_at: Date
 }
 
+const addressCheck = (data: string) => !data || (!!data && isEthereumAddress(data))
+
+export type PersonnelInCreation = Pick<
+  PersonnelAttributes,
+  'name' | 'address' | 'role' | 'about' | 'relevantLink' | 'project_id'
+>
+export const PERSONNEL_IN_CREATION_SCHEMA: ZodSchema<PersonnelInCreation> = z.object({
```

Comment:
We have multiple casings for these schemas, we should define one! We have an `EventFilterSchema` and a `reasonSchema` too.
My vote goes to `PersonnelInCreationSchema`.